package.json

```diff
@@ -93,74 +93,74 @@
     "react-hook-form": "^7.36.0",
     "react-query": "^3.39.2",
     "react-redux": "^8.0.4",
-    "react-router-dom": "^5.2.0",
+    "react-router-dom": "^6.7.0",
     "redux": "^4.2.0",
     "redux-logger": "3.0.6",
     "redux-promise-middleware": "6.1.3",
     "semver": "^7.3.8",
-    "ts-loader": "^9.4.1",
-    "typescript": "^4.8.4",
+    "ts-loader": "^9.4.2",
+    "typescript": "^4.9.5",
     "uuid": "^9.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.16.0",
-    "@babel/plugin-proposal-object-rest-spread": "7.18.9",
+    "@babel/core": "^7.21.0",
+    "@babel/plugin-proposal-object-rest-spread": "7.20.7",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
-    "@babel/plugin-transform-runtime": "7.19.1",
-    "@babel/preset-env": "7.19.3",
+    "@babel/plugin-transform-runtime": "7.21.0",
+    "@babel/preset-env": "7.20.2",
     "@babel/preset-flow": "7.18.6",
     "@babel/preset-react": "7.18.6",
     "@babel/preset-typescript": "^7.18.6",
     "@faker-js/faker": "^7.5.0",
     "@redhat-cloud-services/eslint-config-redhat-cloud-services": "^1.2.4",
-    "@redhat-cloud-services/frontend-components-config": "^4.6.21",
+    "@redhat-cloud-services/frontend-components-config": "^4.7.0",
     "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^12.1.2",
+    "@testing-library/react": "^12.0.0",
```

Comment:
I would leave this line how it was. Both will install 12.x.x on `npm i` but the new change will allow older versions of this package to be supported.